SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesConstants.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialGDKServicesModule.h"
+
+namespace SpatialGDKServicesConstants
+{
+    namespace
+    {
```

Comment:
Anonymous namespace in header file is a bit uncommon. I'd rather remove it, and just use `SpatialGDKServicesConstants` namespace.